### ISSUE TITLE
Don’t use grunt

### ISSUE THREAD

**Issue by mightyiam** [Urucas/sha-bang#1]
Just a suggestion here. It seems to me that grunt is overkill here.

If you do keep it, at least move it to `devDependencies`.

**Comment by vrunoa** [comment]
Yeah i'm not using grunt anymore on my projects but this got outdated. I'll probably update this repo before the end of the year
